src/audiomixerboard.cpp

```diff
@@ -243,8 +259,9 @@ void CChannelFader::SetupFaderTag ( const ESkillLevel eSkillLevel )
 
 void CChannelFader::Reset()
 {
-    // init gain value -> maximum value as definition according to server
+    // init gain and pan value -> maximum value as definition according to server
     pFader->setValue ( AUD_MIX_FADER_MAX );
+    pPan->setValue ( AUD_MIX_FADER_MAX/2 );
```

Comment:
Ok, you can re-use AUD_MIX_FADER_MAX but we may create a new one for pan. But this is low priority.

Reply:
Yes, I agree, for better readability and logic of the code.

Reply:
Did that. I noticed also that the pan states are not stored and restored as volume fader values yet. But maybe it is ok, when new client connects, the pan is always in the centre.

Reply:
The Jamulus users will not like it, I guarentee you ;-). Storing of the pan will later be a must have but for a first start it is ok to not have it.